public/templates/data.tmpl

```diff
@@ -18,13 +18,25 @@
   </style>
   {{/is_vector}}
   {{^is_vector}}
-  <link rel="stylesheet" type="text/css" href="{{public_url}}leaflet.css" />
-  <script src="{{public_url}}leaflet.js"></script>
-  <script src="{{public_url}}leaflet-hash.js"></script>
-  <script src="{{public_url}}L.TileLayer.NoGap.js"></script>
+  <link rel="stylesheet" type="text/css" href="{{public_url}}leaflet.css{{&key_query}}" />
+  <script src="{{public_url}}leaflet.js{{&key_query}}"></script>
+  <script src="{{public_url}}leaflet-hash.js{{&key_query}}"></script>
+  <script src="{{public_url}}L.TileLayer.NoGap.js{{&key_query}}"></script>
   <style>
     body { margin:0; padding:0; }
     #map { position:absolute; top:0; bottom:0; width:100%; }
+    .leaflet-control-layers-toggle {
+      background-image: url({{public_url}}images/layers.png{{&key_query}});
+      width: 36px;
+      height: 36px;
+    }
+    .leaflet-retina .leaflet-control-layers-toggle {
+      background-image: url({{public_url}}images/layers-2x.png{{&key_query}});
+      background-size: 26px 26px;
+    }
+    .leaflet-default-icon-path { /* used only in path-guessing heuristic, see L.Icon.Default */
+      background-image: url({{public_url}}images/marker-icon.png{{&key_query}});
```

Comment:
just an fyi, these image files don't exist

Reply:
oh lol, i thought they'd be in the [images](https://github.com/maptiler/tileserver-gl/tree/master/public/resources/images) directory.

Reply:
In the working screenshot here – https://github.com/maptiler/tileserver-gl/issues/729#issue-1556553595, the http error was 401 instead of 404 so I assumed they'd be there

Reply:
I wonder if we should add them in. I would think the leaflet BSD-2 license would allow us to distribute them...

Our own LICENSE.md doc need updating though.